### PR TITLE
Contenteditable issue

### PR DESCRIPTION
**Pull request by shlajin**
Hello!
I ran into a problem when there is no possibility of making keys work when focus is inside the contenteditable tag. I believe this should come as an option in `allowIn` array, just like `TEXTAREA`, `INPUT` or `SELECT`.

Thanks!

### CODE REVIEW

**Comment by chieffancypants** [comment]
Makes sense -- can you please squash the commits so it's a bit easier to review?

**Comment by chocking-u** [comment]
Sure, just did it.

File: src/hotkeys.js [hunk 1]
```diff
@@ -76,7 +76,7 @@
           return false;
         }
 
-        return (element.contentEditable && element.contentEditable == 'true');
+        return (element.contentEditable && element.contentEditable == 'false');
```
Comment (chieffancypants) [inline src/hotkeys.js]: Won't this always return false?  What's the point in keeping this here at this point?  It seems as though the `stopCallback` should now just always return false

Comment (chocking-u) [inline src/hotkeys.js]: `contentEditable` is a string, not boolean (sounds not too good, but that's how browser returns this value). So yes, probably first clause `element.contentEditable` is extra.

**Comment by chieffancypants** [comment]
Closing due to being out of date with master and no updates after I made some suggestions.  If this is still important to you, please open a new PR that has been updated against master.

**Pull request closed without merging**